model/src/main/proto/topic.proto

```diff
@@ -151,11 +152,18 @@ message CompletedStory {
 
 // Corresponds to the list of stories the player is currently playing across all topics.
 message OngoingStoryList {
```

Comment:
Suggested renaming to `RecommendedActivityList`. 

Also, as per our discussion we need `OngoingStoryList` which will be used in ProfileProgress and RecentlyPlayedActivity and `RecommendedActivityList` will be used in `Home Screen`.

Reply:
Renamed it `RecommendedActivityList`.